caddy/Caddyfile.dev

```diff
@@ -1,32 +0,0 @@
-http://{$SERVER_DOMAIN} {
```

Comment:
It may be possible to use Caddy snippets to further deduplicate Caddy files:
https://gist.github.com/Aivean/05e94e4d5b6bb910f86f76bae354445f

Reply:
I'll take a look. IIRC, snippets are static, can't be parameterized, and that was an issue for further deduplication.

Reply:
they can be conditionally enabled based on env variable:
```
    # Optional TLS configuration based on environment
    @has_ssl {
        environment SSL true
    }
    tls @has_ssl {$SSL_CERT_FILE} {$SSL_KEY_FILE}
```

Reply:
Is that a special plugin?

`caddy validate ./Caddyfile`:

`Error: adapting config using caddyfile: getting matcher module 'environment': module not registered: http.matchers.environment`

Reply:
Plus, `tls` directive does not accept a matcher, so it's unconditional.

Reply:
I also don't see too much value in creating snippets for parts that are used in one place only, like uploading.

Reply:
Updated the dev configuration for frontend / backend, but don't see a way of optimizing TLS configuration this way. Let me know if you have any other ideas.